.chloggen/2528.yaml

```diff
@@ -0,0 +1,8 @@
+change_type: enhancement
+component: gen-ai
+note: | 
+  Add new attributes to 'invoke_agent', 'execute_tool' and llm inference spans to capture input and output definition for Single and Multi-Agent systems.
```

Comment:
```suggestion
  Add tool definitions and execution details:
```

Reply:
@lmolkova : made the changes in latest push. Please review